### PR TITLE
Change to use env files 

### PR DESCRIPTION
**Pull request by jhagberg**
Added env file in docker services.

Update R and python files.

### CODE REVIEW

File: app/utils/settings.py [hunk 1]
```diff
@@ -4,30 +4,16 @@
...
 
-postgres.name = PARSER.get("postgres", "name")
-postgres.host = PARSER.get("postgres", "host")
-postgres.port = PARSER.get("postgres", "port")
-postgres.user = PARSER.get("postgres", "user")
-postgres.password = PARSER.get("postgres", "password")
+#Read configuration from environment variable
+
+postgres.name = os.environ.get("POSTGRES_DB", "herdbook")
+postgres.host = os.environ.get("POSTGRES_host", "herdbook-db")
```
Comment (jonandernovella) [inline app/utils/settings.py]: Should be `postgres.host = os.environ.get("POSTGRES_HOST", "herdbook-db")`

**Review by jonandernovella — CHANGES_REQUESTED** [review]

File: app/utils/settings.py (hunk 1 above)
Comment (jhagberg) [reply app/utils/settings.py]: oh yes

**Review by pontus — APPROVED** [review]

**Review by jonandernovella — APPROVED** [review]
LGTM

**Review by norling — CHANGES_REQUESTED** [review]
This PR breaks our coding guidelines against not having sensitive data in repos. I'd prefer to add `database-variables.env` (and possibly `r-api-variables.env`) to `.gitignore` and allow the system to run on default values - or have a startup-script that creates the environment files with a secure password if the files doesn't exist.
The `README` should also be updated with information on how to change settings in these files, and `init_db.sh` should either be removed or updated as this PR will break that script (I'd be fine with removing it).

**Comment by andkaha** [comment]
> This PR breaks our coding guidelines against not having sensitive data in repos. I'd prefer to add `database-variables.env` (and possibly `r-api-variables.env`) to `.gitignore` and allow the system to run on default values - or have a startup-script that creates the environment files with a secure password if the files doesn't exist.
> The `README` should also be updated with information on how to change settings in these files, and `init_db.sh` should either be removed or updated as this PR will break that script (I'd be fine with removing it).

Using a `.default` file would be in order, IMHO, just like we previously had with `app/config.ini.default`.

**Comment by norling** [comment]
Yes! that would be fine! :) Are you still using `init_db.sh` btw?

**Comment by andkaha** [comment]
> Yes! that would be fine! :) Are you still using `init_db.sh` btw?


Yes, so that script would possibly need to be updated(?)

It should at least contain an updated path to the correct "env file".

**Comment by jhagberg** [comment]
> The `README` should also be updated with information on how to change settings in these files, and `init_db.sh` should either be removed or updated as this PR will break that script (I'd be fine with removing it).

But there is no information now on how to do this in the README so should it be an instruction on how to do it by hand or a init script?

File: README.md [hunk 2]
```diff
@@ -44,6 +53,8 @@ f62cebbc2e43        herdbook_main              "/bin/sh -c /entrypo…"   3 hour
 
 ```
 
+If you 
+
```
Comment (norling) [inline README.md]: Some collaborators who are new to the project may have trouble understanding the nuances of this paragraph.

**Review by norling — APPROVED** [review]

**Pull request merged**